### PR TITLE
CustomInfo Real Fix

### PR DESCRIPTION
**Pull request by louis1706**

### CODE REVIEW

**Comment by louis1706** [comment]
tested Here (https://discord.com/channels/656673194693885975/1107600981144903750/1107682651806126080)

File: Exiled.API/Features/Player.cs [hunk 1]
```diff
@@ -389,26 +385,25 @@ public string CustomInfo
...
-                                Log.Error($"Custom info of player {Nickname} has been REJECTED. reason: (Bad text reject) Info: {value}");
+                                Log.Error($"Custom info of player {Nickname} has been REJECTED. \nreason: (Bad text reject) \ntoken: {token} \nInfo: {value}");
                             else if (!Misc.AllowedColors.ContainsValue(token.Substring(0, 7)))
-                                Log.Error($"Custom info of player {Nickname} has been REJECTED. reason: (Bad colour reject) Info: {value}");
+                                Log.Error($"Custom info of player {Nickname} has been REJECTED. \nreason: (Bad colour reject) \ntoken: {token} \nInfo: {value}");
                         }
                         else
                         {
-                            Log.Error($"Custom info of player {Nickname} has been REJECTED. reason: (Bad text reject) Info: {value}");
+                            Log.Error($"Custom info of player {Nickname} has been REJECTED. \nreason: (Bad colour reject) \ntoken: {token} \nInfo: {value}");
```
Comment (IRacle1) [inline Exiled.API/Features/Player.cs:401]: Colour?

Comment (Misaka-ZeroTwo) [reply Exiled.API/Features/Player.cs:401]: > Colour?

That's British English Lol. And yes, use US English.

Comment (louis1706) [reply Exiled.API/Features/Player.cs:401]: sorry we learned Britsh english at school lol

**Comment by Misaka-ZeroTwo** [comment]
Fix #1741

**Pull request merged**